src/twisted/python/_release.py

```diff
@@ -501,12 +503,22 @@ def main(self, args):
         @param args: The command line arguments to process.  This must contain
             two strings: the path to the root of the Twisted checkout, and a
             path to an output directory.
+
+        @return: The list of build errors.
+        @rtype: List of L{str}
+
         """
         if len(args) != 2:
             sys.exit(
                 "Must specify two arguments: " "Twisted checkout and destination path"
             )
-        self.buildAPIDocs(FilePath(args[0]), FilePath(args[1]))
+
+        capture = StringIO()
+        with redirect_stdout(capture):
+            self.buildAPIDocs(FilePath(args[0]), FilePath(args[1]))
```

Comment:
I think the best long-term solution is to use a logging API in pydoctor, instead of writing to stdout. However, that is not something that needs to happen right now and the stdout capture will still work when pydoctor does make the switch.